src/ripple/app/paths/cursor/Liquidity.cpp

```diff
@@ -45,7 +45,7 @@ TER PathCursor::liquidity () const
             << "reverseLiquidity< "
             << "nodeIndex=" << pc.nodeIndex_
             << " resultCode=" << transToken (resultCode)
-            << " transferRate_=" << pc.node().transferRate_
+            << " transferRate_=" << *pc.node().transferRate_
```

Comment:
This is correct as `transferRate_` is always set in `reverseLiquidity`. However, I don't like that this code will crash if we change that behavior (especially since it's just logging and is easy to overlook in future code reviews). How about:
`pc.node().transferRate_.value_or(invalidRate)`? Here's a commit to cherry pick if you want: https://github.com/seelabs/rippled/commit/de544086690e4a822b3ab9d6a75bb5179059e834 (Also OK as is).